### PR TITLE
Favorite screen: Create UI

### PR DESCRIPTION
**Pull request by nhikiu**
## Related Tickets
- [#Task 78451: Content](https://edu-redmine.sun-asterisk.vn/issues/78451)
## WHAT
- Favourite Screen - Create UI
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

**Review by shinnosuke14202 — APPROVED** [review]

**Review by nang2002hv — APPROVED** [review]

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<resources>
+
+    <style name="ShapeAppearanceOverlay.Small" parent="">
+        <item name="cornerFamily">rounded</item>
+        <item name="cornerFamilyBottomLeft">rounded</item>
+        <item name="cornerFamilyBottomRight">rounded</item>
+        <item name="cornerFamilyTopLeft">rounded</item>
+        <item name="cornerFamilyTopRight">rounded</item>
+        <item name="cornerSize">@dimen/image_corner_cover_favorite_book</item>
+    </style>
```
Comment (ghost) [inline app/src/main/res/values/styles.xml:10]:
```suggestion
    </style>
    
```

File: app/src/main/res/layout/item_favorite_book.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                app:layout_constraintTop_toTopOf="@id/iv_cover" />
+
+            <TextView
+                android:id="@+id/tv_rating"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:ellipsize="end"
+                android:maxLines="1"
+                android:text="0.0"
```
Comment (ghost) [inline app/src/main/res/layout/item_favorite_book.xml]: Nếu là default text, em phải đặt trong string resource.
Nếu là text để test lúc code Ui thì em phải sử dụng tools thay cho android (preview)
```suggestion
                tools:text="0.0"
```

File: app/src/main/java/com/example/book/screen/favorite/FavoritePresenter.kt [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                    "https://www.wilsoncenter.org/sites/default/files/media/images/person/james-person-1.jpg",
+                    4.5,
+                ),
+                Book(
+                    10,
+                    "Title",
+                    "Author",
+                    "Description",
+                    "https://www.wilsoncenter.org/sites/default/files/media/images/person/james-person-1.jpg",
+                    4.5,
```
Comment (ghost) [inline app/src/main/java/com/example/book/screen/favorite/FavoritePresenter.kt:101]: Viết hàm trả về Data fake, không đặt trong onStart em

**Review by ghost — CHANGES_REQUESTED** [review]
Check my cmts

**Review by ghost — APPROVED** [review]

**Pull request merged**